src/czml3/properties.py

```diff
@@ -1,7 +1,47 @@
 from .base import BaseCZMLObject
 from .common import Deletable, HasAlignment, Interpolatable
 from .enums import ClockRanges, ClockSteps, LabelStyles
-from .types import Cartesian3Value, Uri
+from .types import Cartesian3Value, RgbafValue, RgbaValue, Uri
+
+
+class Color(BaseCZMLObject, Interpolatable, Deletable):
+    """A color. The color can optionally vary over time."""
+
+    KNOWN_PROPERTIES = ["delete", "rgba", "rgbaf", "reference"]
+
+    def __init__(self, *, delete=None, rgba=None, rgbaf=None):
```

Comment:
Do we want to allow both `rgba` and `rgbaf`?

Reply:
It is considered valid in Cesium (though technically, if both parameters are defined, only ``rgbaf`` is considered) so I don't see the reason to throw an error

Reply:
Oh, didn't know that! It's not explained here https://github.com/AnalyticalGraphicsInc/czml-writer/wiki/Color perhaps we should add a note to clarify.